app/auth-hooks.js

```diff
@@ -11,6 +12,11 @@ export const requireAuth = (store) => (nextState, replace, done) => {
             pathname: '/login',
             state: { nextPathname: nextState.location.pathname }
           });
+        } else {
+          const socket = notifyHub.getInstance();
+          if (socket.disconnected) {
```

Comment:
`connected` and `disconnected` props are not part of the public API of socket.io and have changed in the past, it's up to you whether you still want to use them but I'd suggest keeping track of the state outside of the socket object and changing it in the event handlers.